plugins/modules/gitlab_project_badge.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Guillaume MARTINEZ (lunik@tiwabbit.fr)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: gitlab_project_badge
+short_description: Manage project badges on GitLab Server
+version_added: 6.0.0
+description:
+  - This module allows to add and remove badges to/from a project.
+author: Guillaume MARTINEZ (@Lunik)
+requirements:
+  - C(owner) or C(maintainer) rights to project on the GitLab server
+extends_documentation_fragment:
+  - community.general.auth_basic
+  - community.general.gitlab
```

Comment:
... do this:
```suggestion
  - community.general.gitlab
  - community.general.attributes

attributes:
  check_mode:
    support: full
  diff_mode:
    support: none
```
This will render as https://docs.ansible.com/ansible/devel/collections/community/crypto/x509_certificate_pipe_module.html#attributes (of course with a different support value for `diff`).

Reply:
corrected by c4856f28490040f4f5274fbc90624ea7841df846